specs/trust-store-trust-policy.md

```diff
@@ -345,11 +344,9 @@ Notary v2 allows user to execute custom validations during verificaton using plu
 
 ### Certificate Revocation Evaluation
 
-This section should be considered as **DRAFT** and will be updated after RC1 release.
-
 If the certificate revocation validation is set to `skip` using policy, skip the below steps, otherwise check for revocation status for certificate and certificate chain.
 
-- If the revocation status of any of the certificates is revoked or cannot be determined (revocation status unavailable), fail this step. This will fail the overall signature verification or log the error, depending on if revocations status check is `enforced` or `logged`, as per the signature verification level.
+If the revocation status of any of the certificates is revoked or cannot be determined (revocation unavailable), fail this step. This will fail the overall signature verification or log the error, depending on if revocations status check is `enforced` or `logged`, as per the signature verification level.
```

Comment:
The first sentence is a bit confusing. Can we provide some more clarity? At the moment is reads:
> If the revocation status of any of the certificates is revoked or cannot be determined (revocation unavailable)
If the _revocation status is revoked_ doesn't this mean that the certificate is valid? I would suggest to reword to make sure we are clear what the behavior is.